ADD New package revocap-refiner

h-denpo opened the pull request:
"""The University of Tokyo, CISS Project:
        Geometric processing, mesh processing, mesh generation"""

h-denpo committed:
ADD New package revocap-refiner
    """The University of Tokyo, CISS Project:
        Geometric processing, mesh processing, mesh generation"""

adamjstewart commented on var/spack/repos/builtin/packages/revocap-refiner/package.py:
```diff
@@ -0,0 +1,58 @@
+    homepage = "https://github.com/FrontISTR/FrontISTR"
+    git      = "https://github.com/FrontISTR/REVOCAP_Refiner.git"
+
+    version('master', commit='626d445')
```
```suggestion
    version('master', branch='master')
```

adamjstewart commented on var/spack/repos/builtin/packages/revocap-refiner/package.py:
```diff
@@ -0,0 +1,58 @@
+    depends_on('ruby', type='build')
+    depends_on('mpi')
+    depends_on('doxygen', type='build')
+    depends_on('swig')
```
```suggestion
    depends_on('swig', type='build')
```

adamjstewart commented on var/spack/repos/builtin/packages/revocap-refiner/package.py:
```diff
@@ -0,0 +1,58 @@
+        install_tree('bin', prefix.bin)
+        install_tree('lib', prefix.lib)
+        mkdirp(prefix.include)
+        install_tree('Refiner', prefix.include)
```
```suggestion
        install_tree('Refiner', prefix.include.Refine)
```

adamjstewart commented on var/spack/repos/builtin/packages/revocap-refiner/package.py:
```diff
@@ -0,0 +1,58 @@
+    def install(self, spec, prefix):
+        install_tree('bin', prefix.bin)
+        install_tree('lib', prefix.lib)
+        mkdirp(prefix.include)
```
I don't think this line is actually needed

h-denpo replied:
I think so, too.

adamjstewart commented on var/spack/repos/builtin/packages/revocap-refiner/package.py:
```diff
@@ -0,0 +1,58 @@
+    """The University of Tokyo, CISS Project:
+        Geometric processing, mesh processing, mesh generation"""
+
+    homepage = "https://github.com/FrontISTR/FrontISTR"
```
Why doesn't the homepage match the same git repository?

h-denpo replied:
I made a mistake in writing.
The correct URL is https://github.com/FrontISTR/REVOCAP_Refiner

h-denpo committed:
Modify     homepage = "https://github.com/FrontISTR/REVOCAP_Refiner"
Modify    version('master', branch='master')
Modify    depends_on('swig', type='build')
Modify        install_tree('Refiner', prefix.include.refine)
Delete         mkdirp(prefix.include)
